.github/workflows/pytest.yml

```
@@ -0,0 +1,35 @@
+name: Django CI with Docker Compose
+
+on:
+  workflow_dispatch:
+  push:
+      branches:
+      - main
```

```suggestion
    branches:
      - main
```

I think there's an indentation error